sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -393,14 +393,6 @@ SUBROUTINE INITPOST_NETCDF(ncid3d)
       else
        if(me==0)print*,'ak5= ',ak5
       end if 
-      Status=nf90_get_att(ncid3d,nf90_global,'sf_surface_physi', &
-             iSF_SURFACE_PHYSICS)
-      if(Status/=0)then
-       print*,'sf_surface_physi not found; assigning to 2'
-       iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
```

Comment:
@EricJames-NOAA These are current setting for EMC fv3lam. You remove them to WRFPOST.f and read iSF_SURFACE_PHYSICS from attribute "landsfcmdl" instead of "sf_surface_physi" for GSL RRFS.  Can we unify this for GSL RRFS and EMC fv3lam? 
@ericaligo-NOAA Do you have comments on that?
Thanks!

Reply:
For EMC fv3lam, that iSF_SURFACE_PHYSICS nf90_get_att does not work, I see this message in the logs:

sf_surface_physi not found; assigning to 2

I can easily modify my PR to similarly always set sf_surface_physics to 2 for EMC fv3lam, but maybe it would be good to read the correct FV3 attribute "landsfcmdl" and figure out why it is set to 1 for EMC fv3lam

Reply:
@ericaligo-NOAA @EricRogers-NOAA  Do you have any comments on landsfcmdl setting as 1 from EMC fv3lam?